examples/unionjack.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+
+import colorsys
+import time
+import blinkt
+
+Red   = (255,0,0)
```

Comment:
Python convention would have these as `RED`, `WHITE` and `BLUE` to indicate constant values. Capitalised names are usually classes.

Reply:
Done